packages/template-retail-react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "2.3.0",
+  "version": "2.3.1-preview.0",
```

Comment:
There's no other package in the monorepo that depends on retail-react-app. So we can release it by itself 👍